config/config.py

```diff
@@ -96,6 +96,11 @@ def desks(self):
 	def __str__(self):
 		return self.__properties.__str__()
 
+	def update_desk_list(self, new_desk):
+		updated_desks = self.__loader.update_desk_list(new_desk)
+		self.__desks = [_DeskConfigDAO(desk_dict) for desk_dict in updated_desks]
```

Comment:
:+1:

Reply:
I can't comment directly on line 85 but this syntax could be used there too I believe.